stl/inc/mutex

```diff
@@ -544,23 +545,47 @@ int __stdcall _Callback_once(void*, void* _Pv, void**) { // adapt call_once() to
 
 template <class _Fn, class... _Args>
 void(call_once)(once_flag& _Flag, _Fn&& _Fx, _Args&&... _Ax) { // call _Fx(_Ax...) once
-    using _Tuple = tuple<_Fn&&, _Args&&..., exception_ptr&>;
-    using _Seq   = make_index_sequence<1 + sizeof...(_Args)>;
+    int pending;
+    bool fallback = false;

```

Comment:
As this is a header, `pending` and `fallback` must be `_Ugly` to avoid macro conflicts.